runtime/replayer.py

```diff
@@ -28,39 +30,32 @@ def start(self):
             replayer.running = True
             replayer.save()
             self.execute(replayer.id)
-    
+
     def execute(self, id):
         xlog = get_logs(self.log.path)
         # t=threading.Thread(target=self.events_list, args=(xlog, id))
         events = self.events_list(xlog, id)
-        
+
     def thread_fun(self, trace, log, id):
         for event in trace:
             replayer = DemoReplayer.objects.get(pk=id)
             if replayer.running:
                 django_rq.enqueue(prepare, event, trace, log, id, self.reg_id, self.class_id, self.log)
-                sleep(randint(5, 10)) 
+                sleep(randint(5, 20))
```

Comment:
this is bad. remove `sleep` unless you have a VERY good reason

Reply:
For the moment we have to let it there, the sleep will affect onlythreads. so it's not affecting the server performance, and it's needed because so the events will not arrive all in one second